### PR TITLE
spirv-fuzz: Support OpNot bit instruction case

### PR DESCRIPTION
**Pull request by andreperezmaselco**
This PR implements the `OpNot` instruction for the
add bit instruction synonym transformation. In addition,
some code improvements have been made.

### CODE REVIEW

File: test/fuzz/transformation_add_bit_instruction_synonym_test.cpp [hunk 1]
```diff
@@ -288,174 +339,627 @@ TEST(TransformationAddBitInstructionSynonymTest, AddBitwiseSynonym) {
...
+        %270 = OpBitFieldInsert %2 %269 %187 %10 %6 ; inserts bit 5
+        %271 = OpBitFieldInsert %2 %270 %190 %11 %6 ; inserts bit 6
+        %272 = OpBitFieldInsert %2 %271 %193 %12 %6 ; inserts bit 7
+        %273 = OpBitFieldInsert %2 %272 %196 %13 %6 ; inserts bit 8
+        %274 = OpBitFieldInsert %2 %273 %199 %14 %6 ; inserts bit 9
+        %275 = OpBitFieldInsert %2 %274 %202 %15 %6 ; inserts bit 10
+        %276 = OpBitFieldInsert %2 %275 %205 %16 %6 ; inserts bit 11
+        %277 = OpBitFieldInsert %2 %276 %208 %17 %6 ; inserts bit 12
+        %278 = OpBitFieldInsert %2 %277 %211 %18 %6 ; inserts bit 13
+        %279 = OpBitFieldInsert %2 %278 %214 %19 %6 ; inserts bit 14
```
Comment (afd) [inline test/fuzz/transformation_add_bit_instruction_synonym_test.cpp:646]: The Windows bots are complaining that this string is too big:

```
transformation_add_bit_instruction_synonym_test.cpp(646): error C2026: string too big, trailing characters truncated
```

Can you split the string literal up?  (Or split this into multiple tests.)

**Review by afd — COMMENTED** [review]
Thanks for this - it looks good except that there is a Windows CI failure due to a massive string.

File: test/fuzz/transformation_add_bit_instruction_synonym_test.cpp (hunk 1 above)
Comment (andreperezmaselco) [reply test/fuzz/transformation_add_bit_instruction_synonym_test.cpp:646]: Sure.

Comment (andreperezmaselco) [reply test/fuzz/transformation_add_bit_instruction_synonym_test.cpp]: Done.

**Review by afd — APPROVED** [review]
Looks great, thanks!

**Pull request merged**